.golangci.yml

```diff
@@ -58,6 +58,7 @@ linters-settings:
       - name: if-return
         disabled: true
       - name: package-comments
+        disabled: true
```

Comment:
I suggest not.

violates Go standards, It's should fix issues, not suppress.

Reply:
> I suggest not.
> 
> violates Go standards, It's should fix issues, not suppress.

agree with your advice, so I decide to enable `package-comments` rule